### ISSUE TITLE
Unexpected behavior when excluding

### ISSUE THREAD

**Issue by evgeny-petukhov** [browserslist/browserslist#246]
```
npx browserslist "last 2 versions, not < 1% in RU"
npx: installed 3 in 2.624s
and_chr 64
and_ff 57
and_uc 11.8
baidu 7.12
chrome 65
chrome 64
edge 16
firefox 59
firefox 58
ie 11
ios_saf 11.0-11.2
op_mini all
op_mob 37
safari 11
```
**Expected behavior**: select 2 versions of all browsers, then remove all browsers that have less than 1% in Russia. So there should not be browsers in the list that has less than 1% of coverage in RU.
**Now**: There are browsers that have less than 1% coverage in RU, for example, Opera

**Comment by ai** [comment]
Fixed:

* a7754dd
* 4cacf60

**Comment by ai** [comment]
Released in 3.2.5
